### PR TITLE
Resolve 'Finish implementing `/conta` front end page/route'

### LINKED ISSUE #44: Finish implementing `/conta` front-end page/route
Users should be able to edit their password, export their account data, log out and also delete their account at `/conta`.

### PR DESCRIPTION
**Pull request by AtilioA**
Users now can:

- Request account data export
- Logout
- Delete their account
- Update password (actually I couldn't make it work :rose:)

This PR also includes minor QoL improvements such as autofocusing the first input, enlarging its clickable area, and auto-filling the email input field (changing email for an account is not planned; that lays some groundwork for it, but we'll leave it as it is).

Resolves #44.

### CODE REVIEW

**Comment by AtilioA** [comment]
pqp

**Comment by AtilioA** [comment]
I couldn't set `autoFocus` on the `Input` component since it would focus the last input that was rendered.

**Comment by AtilioA** [comment]
Somehow, `display: flex` changes from 063814c470e5632c0805c7128d0ae49c07f0c1ce broke the "Criar notificação" buttons on the dashboard, but this is just a matter of overwriting it with local styles.

File: how [hunk 1]
```diff
@@ -0,0 +1,280 @@
+GIT-STASH(1)                                                Git Manual                                                GIT-STASH(1)
```
Comment (Henriquelay) [inline how]: Holy M*ly

File: web/src/pages/Account/index.tsx [hunk 2]
```diff
@@ -26,46 +43,108 @@ export default function Account() {
...
+      );
+      if (userDataString) {
+        const user: Record<string, string> = JSON.parse(
+          userDataString,
+        ) as Record<string, string>;
+        if (user) {
+          const userEmail: string = user.email;
+          return userEmail;
+        }
+        return 'turing@inf.ufes.br';
```
Comment (Henriquelay) [inline web/src/pages/Account/index.tsx]: Why

File: web/src/pages/Account/index.tsx [hunk 3]
```diff
@@ -101,20 +181,37 @@ export default function Account() {
             placeholder="Confirmação de sua nova senha"
           />
 
-          <button type="submit">Atualizar dados</button>
-          <button id="sair" type="submit">
-            Sair
-          </button>
-          <button id="deletar-conta" type="submit">
-            Deletar conta
+          {/* 'Form' Submit */}
```
Comment (Henriquelay) [inline web/src/pages/Account/index.tsx:183]: Really nice to keep this commented like this. GJ

**Comment by Henriquelay** [comment]
LGTM 👍  sem testar dnv

File: web/src/pages/Account/index.tsx (hunk 2 above)
Comment (AtilioA) [reply web/src/pages/Account/index.tsx]: easteregg

Comment (AtilioA) [reply web/src/pages/Account/index.tsx]: Easter egg.

Comment (AtilioA) [reply web/src/pages/Account/index.tsx]: :(

**Pull request merged**